x/slashing/internal/keeper/signing_info.go

```diff
@@ -1,6 +1,9 @@
 package keeper
```

Comment:
should we rename this file to `slash.go` or `punishment.go` as it now contains logic other than the signing info ?

Reply:
Meh, I want to leave `signing_info.go` as-is, but I will move non-related methods to `keeper.go`.